authors/apps/profiles/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from .models import Profile
+
+
+class ProfileSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(source='user.username')
+    bio = serializers.CharField(allow_blank=True, required=False)
+    # image = serializers.SerializerMethodField()
+    image = serializers.CharField(allow_blank=True, required=False)
+
+    class Meta:
+        model = Profile
+        fields = ('username', 'bio', 'image',)
+        read_only_fields = ('username',)
+
+    def get_image(self, obj):
+        if obj.str:
+            return obj.str
```

Comment:
You are not using this function anywhere since you commented out the image field it used as a serializer method. Also, the image field is allowing any kinds of strings since it's declared as **Charfield** yet you specified it to be URLField in your model.
Example;
```
This shouldn't be right as specified in your PR because of the invalid link in your image data.
{
  "user":{
    "username": "nicksbros",
    "email": "some@some.com",
    "bio":"I work at a statefarm i",
    "image": "https://inspector"
    
}
But this is right.
{
  "user":{
    "username": "nicksbros",
    "email": "some@some.com",
    "bio":"I work at a statefarm i",
    "image": "https://inspector.jpg"
    
}
```
I think making use of the serializer method you commented out will solve this problem but you don't include the brackets since it is not taking in any method_name.
So this would be right.
```
image = serializers.SerializerMethodField
```

Reply:
This will work in a way that It won't allow any input from the user if it's not an image object so, for now, this will keep rendering and empty image field regardless of if someone included it in the request body or not. Which case I cant test in postman at this point